From 5be335ceaac0390ecf6bf72c933cecc1e78c546d Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Tue, 11 Sep 2018 01:45:30 +0200
Subject: [PATCH 172/293] xlat: add FAN_ENABLE_AUDIT and FAN_REPORT_TID to
 fan_init_flags

* xlat/fan_init_flags.in (FAN_ENABLE_AUDIT): New constant, introduced
by Linux commit v4.15-rc1~130^2^2~11.
(FAN_REPORT_TID): New constant, introduced by Linux commit
v4.20-rc1~75^2~2.
* NEWS: Mention this.
* tests/fanotify_init.c (main): Update expected output.

Co-Authored-by: Dmitry V. Levin <ldv@altlinux.org>
---
 xlat/fan_init_flags.in | 2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/xlat/fan_init_flags.in b/xlat/fan_init_flags.in
index ca33039..54d681f 100644
--- a/xlat/fan_init_flags.in
+++ b/xlat/fan_init_flags.in
@@ -2,3 +2,5 @@ FAN_CLOEXEC 0x00000001
 FAN_NONBLOCK 0x00000002
 FAN_UNLIMITED_QUEUE 0x00000010
 FAN_UNLIMITED_MARKS 0x00000020
+FAN_ENABLE_AUDIT 0x00000040
+FAN_REPORT_TID 0x00000100
-- 
1.7.12.4

